src/elements.jl

```diff
@@ -0,0 +1,144 @@
+import PeriodicTable
+
+# Alias to avoid similarity of elements and Element in DFTK module namespace
+periodic_table = PeriodicTable.elements
+
+# Data structure for chemical element and the potential model via which
+# they interact with electrons. A compensating charge background is
+# always assumed.
+
+abstract type AbstractElement end
```

Comment:
I liked the name `Element` for this. The fact it's an abstract type doesn't have to show up in the type name, I think it's mostly used for when there's a disambiguation (eg Array vs AbstractArray).

Reply:
ok.